tests/test_capacity_analysis_with_connection_scenario.py

```diff
@@ -97,68 +97,92 @@ def setUpClass(cls) -> None:
 
     def test_capacity_analysis_stats(self) -> None:
         self.assertEqual(0, len(CapacityAnalysisStats.contingencies_dict()))
-        self.assertEqual(17, len(CapacityAnalysisStats.feasibility_dict()))
-        self.assertEqual(
-            [
-                UnfeasibleCondition(
-                    100 + 48.432210483785255j,
-                    LimitingFactor(
-                        Violations.BUS_UNDERVOLTAGE,
-                        ss=None,
+        if sys.platform == "win32" and not envs.pandapower_backend:
+            self.assertEqual(17, len(CapacityAnalysisStats.feasibility_dict()))
+            self.assertEqual(
+                [
+                    UnfeasibleCondition(
+                        100 + 48.432210483785255j,
+                        LimitingFactor(
+                            Violations.BUS_UNDERVOLTAGE,
+                            ss=None,
+                        ),
                     ),
-                ),
-                UnfeasibleCondition(
-                    50 + 24.216105241892627j,
-                    LimitingFactor(
-                        Violations.BUS_UNDERVOLTAGE,
-                        ss=None,
+                    UnfeasibleCondition(
+                        50 + 24.216105241892627j,
+                        LimitingFactor(
+                            Violations.BUS_UNDERVOLTAGE,
+                            ss=None,
+                        ),
                     ),
-                ),
-                UnfeasibleCondition(
-                    25 + 12.108052620946314j,
-                    lf=LimitingFactor(Violations.BUS_UNDERVOLTAGE, ss=None),
-                ),
-            ],
-            CapacityAnalysisStats.feasibility_dict()[
-                Bus(number=152, ex_name="MID500      500.00", type=1)
-            ],
-        )
+                    UnfeasibleCondition(
+                        25 + 12.108052620946314j,
+                        lf=LimitingFactor(Violations.BUS_UNDERVOLTAGE, ss=None),
+                    ),
+                ],
+                CapacityAnalysisStats.feasibility_dict()[
+                    Bus(number=152, ex_name="MID500      500.00", type=1)
```

Comment:
For PandaPower that is
```suggestion
                    Bus(number=151, ex_name="500.0 1.0", type=1)
```